Imp: Throw an early error at submittion time when invalid number of resources was requested (#230)
zapatacomputing/orquestra-workflow-sdk: CHANGELOG.md

💅 *Improvements*
* `sdk.current_run_ids()` now returns a `NamedTuple` called `CurrentRunIDs` to help with typing.

> * Task that requests ray-incompatible number of resources throws an error at submission time

Reviewer: ```suggestion
* Tasks that request resources that are incompatible with Ray will throw an error at submission time.
```

> * Task that requests ray-incompatible number of resources throws an error at submission time

Reviewer: <!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏What about workflow resources?

Author: Honestly - im not super confident on how those are parsed. Workflow resources are not things we pass to ray directly. Im gonna start slack thread if there are any limitations on workflow resources
